### ISSUE TITLE
tfx returns error: TypeError: Cannot redefine property: length

### ISSUE THREAD

**Issue by IvanBoyko** [microsoft/tfs-cli#249]
I'm using a fresh nodejs-lts install:
npm: '5.6.0'

I run installation according to documentation: `npm install -g tfx-cli`
and it succeeds.

However, when I try to run it, it returns:

> error: TypeError: Cannot redefine property: length

**Comment by kxnaylorCLE216** [comment]
I had the same issue until I uninstalled and reinstalled node.js. When reinstalling node.js I used Chocolatey by using this command "cinst nodejs.install"

**Comment by IvanBoyko** [comment]
Hmm... tried this but it didn't work for me. Thanks anyway.

**Comment by bangaruchary** [comment]
Hi, I am using NPM 6.1.0 version. I got the same error. Even installing node.js through chocolatey didn't solve the issue. Please suggest. 
--Chary

**Comment by soccerjoshj07** [comment]
I had this issue because I had a super old version of Node.js remaining, even though there was no mention of it in add/remove programs. I deleted this directory and removed from the system PATH variable:
`C:\Program Files (x86)\Nodejs`

I knew something like this was an issue b/c when I ran `node -v` from a command line, it showed something really old like 0.12.0. The new version of node installed in `C:\Program Files\Nodejs` for me.

More details:
https://stackoverflow.com/questions/20711240/how-to-completely-remove-node-js-from-windows
